f8a_tagger/parsers/parsers.py

```diff
@@ -40,6 +40,20 @@ def parse(self, content):
         return BeautifulSoup(markdown2.markdown(content), 'lxml').get_text()
 
 
+class HtmlParser(AbstractParser):  # pylint: disable=too-few-public-methods
+    """HTML parser."""
+
+    def parse(self, content):
+        """Parse content to raw text.
+
+        :param content: content to parse
+        :type content: str
+        :return: raw/plain content representation
+        :rtype: str
+        """
+        return BeautifulSoup(content).get_text()
```

Comment:
using `BeautifulSoup(content)` gives a warning, does it not?

Reply:
Nice catch! Updated.